app/client-app/src/app/pages/Wordlist/index.jsx

```diff
@@ -97,6 +113,70 @@ const Wordlist = () => {
             filter={filter}
             setFilter={setFilter}
           />
+
+          <HStack spacing={5} align="center">
+            <FormLabel mb="0" mr={2} htmlFor="word-length">
+              Word Length:{" "}
+            </FormLabel>
+            <Box width="300px">
+              <RangeSlider
+                id="word-length"
+                defaultValue={[1, 15]}
+                min={1}
+                max={15}
+                step={1}
+                onChange={handleSliderChange}
+                value={sliderValues}
+              >
+                <RangeSliderTrack bg="blue.100">
+                  <RangeSliderFilledTrack bg="blue.500" />
+                </RangeSliderTrack>
+                <Tooltip label={String(sliderValues[0])} placement="top">
+                  <RangeSliderThumb boxSize={4} index={0} />
+                </Tooltip>
+                <Tooltip label={String(sliderValues[1])} placement="top">
+                  <RangeSliderThumb boxSize={4} index={1} />
+                </Tooltip>
+                <Box
```

Comment:
Is it possible to use the slider marks component instead of building this out custom?
https://chakra-ui.com/docs/components/slider/usage#slider-with-custom-labels-and-marks

Reply:
or actually see: https://github.com/chakra-ui/chakra-ui/discussions/5349

still not documented, but is in the code:
https://github.com/chakra-ui/chakra-ui/blob/1a97c08b5bc159dfa17268a88e3a6a5c4ece339c/packages/components/slider/src/range-slider.tsx#L173

Reply:
I was sure there was a documented way, but we didn't find it in the range slider docs. Didn't think to look on regular slider!

Definitely preferable.